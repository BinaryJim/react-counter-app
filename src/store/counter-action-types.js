export const counterActionTypes = {
  increment: "increment",
  decrement: "decrement",
  reset: "reset",
};
