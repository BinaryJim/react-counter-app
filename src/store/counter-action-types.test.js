import { counterActionTypes } from "./counter-action-types"

describe("counterActionTypes", () => {
  it("should match snapshot", () => {
    expect(counterActionTypes).toMatchSnapshot()
  })
})