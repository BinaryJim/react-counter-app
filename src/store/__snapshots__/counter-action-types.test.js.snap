// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`counterActionTypes should match snapshot 1`] = `
Object {
  "decrement": "decrement",
  "increment": "increment",
  "reset": "reset",
}
`;
